src/lib/blkdata_svc/blkdata_service.cpp

```diff
@@ -55,6 +55,24 @@ void BlkDataService::create_vdev(uint64_t size, HSDevType devType, uint32_t blk_
                                                         .context_data = vdev_ctx.to_blob()});
 }
 
+void BlkDataService::create_vdev_with_chunk_size(uint64_t size, HSDevType devType, uint32_t blk_size, blk_allocator_type_t alloc_type,
```

Comment:
why we want separate function instead of passing both chunk_size and num_chunks in? 

in `create_vdev` we check one and only one is not zero